### ISSUE TITLE
Special Treatment of C13

### ISSUE THREAD

**Issue by sahil21** [ElucidataInc/ElMaven#27]
Reported by Victor on Trello

I was looking at other places where pullIsotopes is used. One of those is in preparing the stacked barplot that shows the isotopic distribution in the EIC window. As expected, with the current bug, where the main window settings aren't read at all, changing the options has no effect. But when those lines are uncommented out (or in Maven 772/775), you can control which isotopes show up in that plot by checking/unchecking them in the options. Except: C13 will always show up, regardless of the option.

This doesn't happen in the plot generation, it happens further upstream when the peakgroup is created. I haven't quite tracked it down yet, but it happens also through the peak detection dialog.

Basically, Maven has all these built-in features to deal with natural 13C abundance (but not natural abundance of anything else), and so there are places where 13C gets special treatment. Eventually, I would like to expose these and make the behavior consistent.

**Comment by GeorgeSabu** [comment]
#43 This bug seems to be solved

**Comment by chubukov** [comment]
Confirming that deselecting 13C works fine for the purposes of the barplot and the peak table. 

We may revisit special treatment of 13C when we look at "auto detect and ignore isotopes" and the NA correction.
